src/main/scala/akka/io/AsyncDnsResolver.scala

```diff
@@ -123,8 +145,13 @@ class AsyncDnsResolver(cache: SimpleDnsCache, config: Config) extends Actor with
 object AsyncDnsResolver {
   private val inetSocketAddress = """(.*?)(?::(\d+))?""".r
 
+  case class SrvResolved(name: String, srv: immutable.Seq[SRVRecord])
```

Comment:
Would you consider renaming to `SRVResolved` to match other SRV related class names?